sdk/keyvault/azure-keyvault-administration/tests/test_access_control.py

```diff
@@ -6,45 +6,45 @@
 import time
 import uuid
 
-from azure.keyvault.administration import KeyVaultRoleScope, KeyVaultPermission, KeyVaultDataAction
+import pytest
+from azure.keyvault.administration import KeyVaultDataAction, KeyVaultPermission, KeyVaultRoleScope
+from devtools_testutils import add_general_regex_sanitizer, recorded_by_proxy, set_bodiless_matcher
 
 from _shared.test_case import KeyVaultTestCase
-from _test_case import AdministrationTestCase, access_control_client_setup, get_decorator
-
+from _test_case import KeyVaultAccessControlClientPreparer, get_decorator
 
 all_api_versions = get_decorator()
 
 
-class AccessControlTests(AdministrationTestCase, KeyVaultTestCase):
-    def __init__(self, *args, **kwargs):
-        super(AccessControlTests, self).__init__(*args, match_body=False, **kwargs)
-
+class TestAccessControl(KeyVaultTestCase):
     def get_replayable_uuid(self, replay_value):
         if self.is_live:
             value = str(uuid.uuid4())
-            self.scrubber.register_name_pair(value, replay_value)
             return value
         return replay_value
 
     def get_service_principal_id(self):
         replay_value = "service-principal-id"
         if self.is_live:
             value = os.environ["AZURE_CLIENT_ID"]
-            self.scrubber.register_name_pair(value, replay_value)
             return value
         return replay_value
 
-    @all_api_versions()
-    @access_control_client_setup
-    def test_role_definitions(self, client):
+    @pytest.mark.parametrize("api_version", all_api_versions)
+    @KeyVaultAccessControlClientPreparer()
+    @recorded_by_proxy
+    def test_role_definitions(self, client, **kwargs):
+        set_bodiless_matcher()
         # list initial role definitions
         scope = KeyVaultRoleScope.GLOBAL
         original_definitions = [d for d in client.list_role_definitions(scope)]
         assert len(original_definitions)
 
         # create custom role definition
+        variables = kwargs.pop("variables")
```

Comment:
Was this part of an effort to use `variables` to store UUIDs?

Reply:
yes it was, forgot to remove it after I added the regex_scrubber and just used the dummy values